### PR TITLE
Updated Turkish Translation.

### PR DESCRIPTION
**Pull request by WeirdYunus**
Mainly updated the UI and some first story text, making it more understandable for turkish players compared to the previous translation. Fixed some wrong translations from the previous one. The amount of lines changed is a bit above the limit but that's mainly because most strings are using space or \n and I can't see the line count on the translation software. I'll try to be more careful next time.

### CODE REVIEW

File: files/lang/tr.po [hunk 1]
```diff
@@ -16,195 +16,220 @@ msgstr ""
...
-"X-Generator: Poedit 3.1.1\n"
+"X-Generator: Poedit 3.5\n"
 
 msgid ""
 "BATTLE\n"
 "ONLY"
 msgstr ""
 "SADECE\n"
-"SAVAŞ"
+"SAVAS"
```
[the remark this answers is not in the document]
Comment (WeirdYunus) [reply files/lang/tr.po:26]: The lines with Capital letters don't support some letters like "Ğ,Ü,Ç,İ,Ş,Ö". That only goes for the UI and some buttons. The regular text font supports those letters. Yeah that would be awesome if we can get support for those letters :). I'm fixing those lines then.

**Comment by WeirdYunus** [comment]
I hope everything is alright now. Don't really know how to use Github properly x)

File: files/lang/tr.po [hunk 2]
```diff
@@ -2779,249 +2915,265 @@ msgid "Need: "
...
-msgstr ""
+msgstr "Etkinlik: "
 
 msgid "Hot Keys:"
-msgstr ""
+msgstr "Kısayollar:"
 
 msgid "Evil"
-msgstr ""
+msgstr "Şeyrani"
```
[the remark this answers is not in the document]
Comment (WeirdYunus) [reply files/lang/tr.po:3090]: Yep typo

**Review by ihhub — APPROVED** [review]

**Comment by ihhub** [comment]
@WeirdYunus , thank you so much for your contribution to the project!

**Pull request merged**